apps/designer/app/designer/features/inspector/inspector.tsx

```diff
@@ -37,13 +39,17 @@ export const Inspector = ({ publish }: InspectorProps) => {
   }
 
   return (
-    <Tabs defaultValue="style" css={{ width: "100%", gap: "$2" }}>
+    <Tabs defaultValue="style" css={{ width: "100%" }}>
```

Comment:
Nit: I wonder if this width was supposed to be defined by the parent container layout ...